test/azure-pipelines.yml

```diff
@@ -154,14 +154,21 @@ jobs:
 
   steps:
   - script: |
-      curl -L https://raw.githubusercontent.com/InsightSoftwareConsortium/ITKPythonPackage/master/scripts/dockcross-manylinux-download-cache-and-build-module-wheels.sh -O
-      chmod u+x dockcross-manylinux-download-cache-and-build-module-wheels.sh
-    displayName: 'Fetch build script'
+      export ITK_PACKAGE_VERSION=$(ITKPythonGitTag)
+      ./wrapping/dockcross-manylinux-download-cache.sh
+    displayName: 'Download ITK build cache'
 
   - script: |
-      export ITK_PACKAGE_VERSION=$(ITKPythonGitTag)
-      ./dockcross-manylinux-download-cache-and-build-module-wheels.sh
-    displayName: 'Build Python packages'
+      ./wrapping/ITKRepositoryGPUPackagesSetup.sh
+    displayName: 'Setup build cache for GPU package builds'
+
+  - script: |
+      ./wrapping/ITKRepositoryGPUPackagesSetup.sh
+    displayName: 'Setup build cache for GPU package builds'
```

Comment:
This script invocation is duplicated. Unintentional?

Reply:
Yes, good catch!